src/3D/Water.cpp

```diff
@@ -28,16 +28,6 @@ Water::Water()
 	_reflectionFrameBuffer =
 	    std::make_unique<FrameBuffer>("Reflection", static_cast<uint16_t>(1024), static_cast<uint16_t>(1024),
 	                                  graphics::Format::RGBA8, graphics::Format::Depth24Stencil8);
-
-	// water texture (256x256 RAW RGB 24bpp)
-	auto& filesystem = Game::instance()->GetFileSystem();
-	auto const& waterTextureData = filesystem.ReadAll(filesystem.TexturePath() / "Sky.raw");
-	const uint16_t textureWidth = 256, textureHeight = 256;
-
-	_texture = std::make_unique<Texture2D>("Water");
-	_texture->Create(textureWidth, textureHeight, 1, Format::RGB8, Wrapping::Repeat, waterTextureData.data(),
```

Comment:
Same issue here. The water plane has no texture and the above differs from the texture loader

![image](https://user-images.githubusercontent.com/32263167/165417021-a4082661-377a-41b3-8b07-0b11be85c41b.png)